utilities/check_mrTrix_Version.m

```diff
@@ -0,0 +1,19 @@
+function mrTrixVersion = check_mrTrix_Version
```

Comment:
Just committed the modified version, please check.
By the way, this doesn't fix it in Red Hat, I need the solution I sent before...